artemis-distribution/src/main/resources/bin/artemis

```diff
@@ -91,6 +91,9 @@ if $cygwin ; then
 fi
 
 exec "$JAVACMD" $JAVA_ARGS $ARTEMIS_CLUSTER_PROPS \
+    --add-opens java.base/java.lang=ALL-UNNAMED \
+    --add-opens java.base/java.io=ALL-UNNAMED \
+    --add-opens java.base/java.util=ALL-UNNAMED \
```

Comment:
hmmmmmmm... I don't need these with JDK17.

@gnodet it would be so nice to get your hep here? :)

Reply:
@gnodet I"m adding jline3 into artemis, integrated with pico-CLI. This PR is taking an example from the jline3 along with some examples I saw on pico-cli .. and I had to include these with JDK-11, while these are not needed in 17.


I'm confused what I need to do to integrate this. can you help us here?

Reply:
Its not actually surprising that behaviour here might differ on different JDKs, thats exactly why I wondered if you tried it on the newer ones, especially with JDK21 almost done (RC1 due in 10 days, later RC2 if needed; JDK17 didnt get an RC2)...though I was more concerned it may need other config, than not need any, we will need to use what works for them all.

Reply:
See above, adding jansi will remove the need for those options.